vmm/src/lib.rs

```diff
@@ -276,9 +276,10 @@ pub fn start_vmm_thread(
         })
         .map_err(Error::VmmThreadSpawn)?;
 
-    // The VMM thread is started, we can start serving HTTP requests
-    api::start_http_thread(http_path, http_api_event, api_sender, seccomp_action)?;
-
+    if let Some(http_path) = http_path {
```

Comment:
Do we need some documentation change since we no longer always start the http server?

Reply:
The HTTP server without a reachable path was inaccessible anyway.